crawler/parlamentares/coautorias/coautorias_tereza_cristina.R

```diff
@@ -14,7 +14,12 @@ proposicoes <- fetch_proposicoes_por_autor(178901)
 
 relacionadas <- fetch_all_relacionadas(proposicoes$id)
 
-autores <- fetch_all_autores(relacionadas, parlamentares)
+autores <- fetch_all_autores(relacionadas, parlamentares) %>%
```

Comment:
Não consegui executar. A mensagem de erro é:
`Error in fetch_all_autores(relacionadas, parlamentares) : 
  unused argument (parlamentares)`
Estou com uma versão desatualizada?